mftf/2.3/getting-started.md

```diff
@@ -204,35 +198,110 @@ vendor/bin/mftf run:test AdminLoginTest --remove
 
 See more commands in [`mftf`][].
 
-### Step 9. Generate reports    {#reports}
+### Step 7. Generate reports    {#reports}
 
 During testing, the MFTF generates test reports in CLI.
 You can generate visual representations of the report data using [Allure Framework][].
 To view the reports in GUI:
 
--  [install Allure][]
--  run the tool to serve the artifacts in `dev/tests/acceptance/tests/_output/allure-results/`:
+- [install Allure][]
+- run the tool to serve the artifacts in `dev/tests/acceptance/tests/_output/allure-results/`:
 
 ```bash
 allure serve dev/tests/acceptance/tests/_output/allure-results/
 ```
 
 Learn more about Allure in the [official documentation][allure docs].
 
+## Set up a standalone MFTF
+
+The MFTF is a root level Magento dependency, but it is also available for use as a standalone application.
+The potential use case is when you develop the MFTF or contribute to it, as it facilitates debugging and tracking changes.
+These guidelines demonstrate how to set up and run Magento acceptance functional tests using standalone MFTF.
+
+### Prerequisites
+
+This installation requires a local instance of the Magento application.
+The MFTF uses the [tests from Magento modules][mftf tests] as well as the `app/autoload.php` file.
+
+### Step 1. Clone the MFTF repository
+
+If you develop or contribute to the MFTF, it makes sense to clone your fork of the MFTF repository.
+For contribution guidelines, refer to the [Contribution Guidelines for the Magento Functional Testing Framework][contributing].
+
+### Step 2. Install the MFTF
+
+```bash
+cd magento2-functional-testing-framework
+```
+
+```bash
+composer install
+```
+
+### Step 3. Build the project   {#build-project}
+
+```bash
+bin/mftf build:project
+```
+
+### Step 4. Edit environment settings
+
+In the `dev/.env` file, define the [basic configuration][] and [`MAGENTO_BP`][] parameters.
+
+### Step 5. Enable the Magento CLI commands {#add-cli-commands}
+
+Copy the `etc/config/command.php` file into your Magento installation at `<magento root directory>/dev/tests/acceptance/utils/`.
+Create the `utils/` directory, if you didn't find it.
+
+### Step 6. Remove the MFTF package dependency in Magento
+
+The MFTF uses the Magento `app/autoload.php` to read the Magento modules.
```

Comment:
Suggested edit: The MFTF uses the Magento `app/autoload.php` file to read Magento modules.

Reply:
:ok: